setup.py

```diff
@@ -29,7 +29,7 @@
     "diskcache>=4",
     "iso8601>=0.1.10; python_version<='3.6'",
     "Jinja2>=2.10.1",
-    "jsonschema>=3.0.2",
+    "jsonschema>=3.0.2,<4",
```

Comment:
Is there a bug filed to come back and look at supporting the jsonschema 4 API at some point?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1713314